Jenkinsfile

```diff
@@ -28,51 +27,18 @@ pipeline {
       }
     }
 
-    stage('NPM Install') {
-      agent {
-        docker {
-          image 'node:16.13.1'
-          reuseNode true
-        }
-      }
-      steps {
-        sh 'yarn install'
-      }
-    }
-
-    stage('Build Production') {
+    stage('Lint and test') {
       agent {
         docker {
           image 'node:16.13.1'
```

Comment:
This NodeJS version is old as hell 😱 Not blocking the PR, but removing the agent block and using any kind of agent (either container `maven-21` or VM with `docker`)  would provide modern NodeJS LTS (20.18.x right now).